Box_intersection_d/include/CGAL/box_intersection_d.h

```diff
@@ -25,12 +25,168 @@
 #include <CGAL/Box_intersection_d/Box_traits_d.h>
 #include <CGAL/Box_intersection_d/box_limits.h>
 
+#include <CGAL/use.h>
+#include <CGAL/tags.h>
+
+#ifdef CGAL_LINKED_WITH_TBB
+#include <tbb/parallel_invoke.h>
+#endif
+
+#include <iterator>
 #include <vector>
+#include <thread>
 
```

Comment:
That is probably the first PR that includes both TBB and `<thread>` from C++11. It seems TBB-4.3 has a bug, and fails to detect that gcc-6.3 has the header `<thread>`. And thus `<tbb/tbb_config.h>` defines `TBB_IMPLEMENT_CPP0X`, and defines its own conflicting version of `std::thread` in `<tbb/compat/thread>`.

Let's ignore that.

Reply:
I don't think the authors meant to include that header. @MaelRL ?

Reply:
It's temporary (I was just curious if I could get the `n` and then find `i`, `j` such that `n=ij` and split in `i` and `j` parts respectively). It will be removed when I finalize the PR.